aptos-move/framework/src/natives/string_utils.rs

```diff
@@ -18,6 +18,7 @@ use move_core_types::{
 use move_vm_runtime::native_functions::NativeFunction;
 use move_vm_types::{
     loaded_data::runtime_types::Type,
+    value_serde::FunctionValueExtension,
```

Comment:
just asking bc imports and no other changes, but if it's not needed linter would complain anyway.

Reply:
There is a line in the file where
```rust
let data = context
                .context
                .function_value_extension()
                .get_serialization_data(fun.as_ref())?;
```

Before, context used here created a `&dyn FunctionValueExtension`, so no import needed. Now an adapter struct is created, so I need to import the trait separately